packages/atlas-service/src/renderer.ts

```diff
@@ -4,6 +4,16 @@ import { ipcInvoke } from './util';
 export function AtlasService() {
```

Comment:
not blocking,  this one kinda confuses me when i see it in the rest of the code, looks like a class name and has the same name of AtlasService (the `main` process class), could we disambiguate it and also have a separate type name for the return type?

For example, probably i would have easier life to look at the client code with something like this:

```   
export function getAtlasServiceRenderer(): ... { ... }

export type AtlasServiceRenderer = ReturnType<getAtlasServiceRenderer>;
```

Reply:
Yeah, I was going back and forth on this wrt interface, couldn't decide what I want (I kinda like the idea of having both as classes, but I definitely should've made `ipcInvoke` to return a constructor then, not a factory). I'll update, but if you don't mind as a follow-up as I already have the incoming PR with opt-in modal that is based on this branch and I don't want to rebase 😅